pyproject.toml

```diff
@@ -103,3 +103,9 @@ force_grid_wrap = 0
 use_parentheses = true
 ensure_newline_before_comments = true
 indent = "\t"
+
+[tool.bench.dev-dependencies]
+coverage = "5.5"
```

Comment:
```suggestion
coverage = "~=6.4.1"
```

Dont see any breaking changes https://coverage.readthedocs.io/en/6.4.1/changes.html 


(haven't checked other deps, don't matter that much)